packages/components/addon-main.cjs

```diff
@@ -4,4 +4,19 @@
  */
 
 const { addonV1Shim } = require('@embroider/addon-shim');
-module.exports = addonV1Shim(__dirname);
+const flightIconSprite = require('@hashicorp/flight-icons/svg-sprite/svg-sprite-module');
+
+module.exports = {
+  ...addonV1Shim(__dirname),
+  contentFor(type, config) {
+    if (
+      !config.emberFlightIcons?.lazyEmbed &&
+      !config.__flightIconsSpriteLoaded &&
```

Comment:
note: I assume the double underscore notation is used to denote the 'internal' nature of this property. across the codebase we have all sorts of notations for this, from `__PROPERTY__` to `_property` and `__property` that would ideally get aligned in the future

Reply:
What do you think the best way to document this would be?